### PR TITLE
KBI0020: Installing DataLad and dependencies in Google Colab

### PR DESCRIPTION
**Pull request by jsheunis**
Closes https://github.com/psychoinformatics-de/knowledge-base/issues/42

### CODE REVIEW

File: kbi/0020/index.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. index::
+   single: datalad;installation
+
+KBI0020: Installing DataLad and dependencies in Google Colab
+============================================================
+
+:authors: Stephan Heunis <jsheunis@gmail.com>
+:discussion: <link>
+:keywords: workaround, credentials, interactive authentication prompt, keyring
```
Comment (mslw) [inline kbi/0020/index.rst:9]: Looks like they got copied from a previous KBI :stuck_out_tongue_winking_eye: feel free to adjust.
```suggestion
:keywords: Google Colab, Conda, installation```

File: kbi/0020/index.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+:software-versions: datalad_0.18.4, git-annex_10.20230330-g98a3ba0ea, conda_23.3.1
+
+Google Colaboratory allows anybody to write and execute arbitrary python code
```
Comment (mslw) [inline kbi/0020/index.rst:12]:
```suggestion
Google Colaboratory allows anybody to write and execute arbitrary Python code
```

**Review by mslw — COMMENTED** [review]
Two general comments:
- In the issue you write that this installs "not permanently" which is understandable, but maybe worth adding what's the scope here (single notebook? single session? I don't know how Collab works so I'm not even sure what could be the options).
- In the notebook you reference a blog post. Would it be useful to reference it in KBI too?

Apart from my comments, this is a nice KBI, and I was able to complete the provided notebook fairly quickly :shipit:

**Comment by jsheunis** [comment]
Thanks @mslw good suggestions!

**Pull request merged**